backend/homework/5/src/main/java/handson/partfive/ProcessingLogs.java

```diff
@@ -0,0 +1,12 @@
+package handson.partfive;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ProcessingLogs {
+    private static final Logger logger = LoggerFactory.getLogger(ProcessingLogs.class);
+
+    public void displayLogs(String msg){
+        logger.info(msg);
+    }
```

Comment:
this logger class is tightly coupled to info level, better use this `getLogger`  inside the specific class whereever needed